### ISSUE TITLE
Reading decimal numbers in Spanish.

### ISSUE THREAD

**Issue by Christianlm** [espeak-ng/espeak-ng#259]
Espeak-ng reads decimal number as  singol digits in spanish, e.g.
2,123 is pronounced [d'os koma 'uno D'os t**'es].

I found a better solution in espeak-ng source:

1. If digital  numbers <= to 5, espeak-ng reads the decimal numbers as entire number.
2. If decimals are more than 5 espeak-ng pronounces digit to digit.

Thought?

**Comment by rhdunn** [comment]
According to the code, there are 3 relevant flags for `tr->langopts.numbers:

*  `NUM_DFRACTION_1` -- adds "hundredths" when given a leading 0 for a 2-digit fraction part. This is used by Italian.
*  `NUM_DFRACTION_2` -- pronounces the fraction as a number, with "zero" for any leading zeros., with a maximum decimal count of 2 The code says that this is the French/Polish behaviour, but is also used by Portuguese.
*  `NUM_DFRACTION_4` -- as `NUM_DFRACTION_2`, but with a maximum decimal count of 5.

I would say that Spanish and Portuguese should have the same behaviour as each other, as they are similar languages and am happy for them both to use `NUM_DFRACTION_4`. I am not sure about Aragonese, Catalan, Interlingua and Papiamento (which share the same base settings as Spanish).

**Comment by Christianlm** [comment]
Hi Reece, I'm not sure about Interlingua language. I consider it as a artificial language and I think we can use the same behaviour of romance languages for read decimals.
 
Ok  for other languages, we can use `NUM_DFRACTION_4` for Aragonese, Catalan, and Papiamento such as Spanish. It seems isn't a linguistic problem.

About `NUM_DFRACTION_1`, it is excellent when there are three digits, but if there are more than five digits espeak-ng pronounces only the last four digits.
I noticed this behaviour time ago and I have fixed it by a regexpt in my screen reader, but a good solution for our mathematicians friends is use `NUM_DFRACTION_4` also for italian language.

Thanks.
